packages/api/src/external/location/address.ts

```diff
@@ -0,0 +1,41 @@
+import { Address } from "@metriport/api-sdk";
+import {
+  getCoordinatesFromLocation,
+  makeLocationClient,
+} from "@metriport/core/external/aws/location";
+import { Config } from "../../shared/config";
+import { Coordinates } from "@metriport/core/external/aws/location";
+
+type AddressWithoutCountry = Omit<Address, "country"> & { country?: string | undefined };
+
+/**
+ * Geocodes a list of addresses using Amazon Location Services.
+ * @param addresses
+ * @returns
+ */
+export async function getCoordinates(addresses: AddressWithoutCountry[]): Promise<Coordinates[]> {
```

Comment:
a little confusing that there is is another getCoordinates function in `parse-cq-directory-entry.ts`. Maybe change this to `getCoordinatesFromAws`?

Reply:
ya maybe ill just change the func name altogether

Reply:
i dont think complicating func names with things like "FromAWS" is a good idea though, but i might be wrong